src/Service/EntityMetadata.php

```diff
@@ -361,4 +379,24 @@ protected function findIliosDtos(KernelInterface $kernel): array
 
         return [...$dtos, ...$classes];
     }
+
+    /**
+     * Get an Entity class name for each DTO type
+     */
+    protected function getEntitiesForDtoTypes(): array
+    {
+        $rhett = [];
+        foreach ($this->iliosDtos as $className) {
+            $ref = new ReflectionClass($className);
+            $type = $this->extractType($ref);
+
+            $name = substr($className, 15, -3);
+            $class = "App\\Entity\\${name}";
```

Comment:
please promote `App\Entity` to be a class constant for higher visibility

Reply:
I'm going to need convincing on this one, `App\Entity` is the namespace for entities and if we change it at some point we'll get an obvious failure here which won't be any easier to find with a constant. Promoting it just makes this less readable in my mind.